docs/user/README.md

```diff
@@ -92,7 +92,13 @@ to load path and require it in `init.el`
 ## Vim and NeoVim
 
 * Install coc.nvim by following the instructions at [coc.nvim]
+  - You will need nodejs installed.
```

Comment:
Why would one need nodejs in this case?

Reply:
Ah, so coc itself is implemented in nodejs. I have no idea how that actually works.

Reply:
Yes, I believe the client-side of the LSP protocol in coc is implemented in js or something. Either way, their installation instruction require it: https://github.com/neoclide/coc.nvim/wiki/Install-coc.nvim